include/binHandler.hpp

```diff
@@ -16,8 +16,8 @@
 #ifndef BINHANDLER_H
 #define BINHANDLER_H
 
-//Define how large the RAM byte array while dumping should be. (100MiB)
-#define _def_ARR_SIZE 104857600
+//Define how large the RAM byte array while dumping should be. (4k)
```

Comment:
If you can prove this is faster on windows + linux, preferably with older hardware (circa 2011 ish) then I'm happy to merge this change.